performance/venv.py

```diff
@@ -240,4 +240,4 @@ def create_virtualenv(python):
 def exec_in_virtualenv(options):
     venv_python = create_virtualenv(options.python)
     args = [venv_python, "-m", "performance"] + sys.argv[1:] + ["--inside-venv"]
-    os.execv(args[0], args)
+    run_cmd(args)
```

Comment:
>It avoids to have two running processes. Using os.execv(), you only get one process. It might help to get more reliable benchmarks.

Seems kinda contrived but may as well be cautious as far as benchmarking goes.